src/cli/commands/public-cmds/init-cmd.ts

```diff
@@ -91,7 +95,8 @@ export default class Init implements LegacyCommand {
         };
       });
     }
-    if (reset && resetHard) throw new GeneralError('please use --reset or --reset-hard. not both');
+    if (reset && resetHard)
+      throw new GeneralError('cannot use both --reset or --reset-hard, please use only one of them');
```

Comment:
I guess it should be "--reset and --reset-hard" (not "or")

Reply:
right! changed